server/models/postgis/task.py

```diff
@@ -142,6 +143,7 @@ class Task(db.Model):
     x = db.Column(db.Integer)
     y = db.Column(db.Integer)
     zoom = db.Column(db.Integer)
+    extra_properties = db.Column(db.Unicode)
```

Comment:
Note: this column already exists in the database ([added through the migrations](https://github.com/hotosm/tasking-manager/blob/56d1d13056bbea342132cf6e78b12f01c73d5dd3/migrations/versions/7d55a089b5bc_.py)), but it wasn't mapped into the DB models.